app/services/kenui/email_notification_service.rb

```diff
@@ -0,0 +1,27 @@
+module Kenui
+  # class where email notification services resides
+  class EmailNotificationService
+    class << self
+
+      def set_email_notification(account_id, is_notified, user = 'email_notification_ui',
+                                 reason = nil, comment = nil, options = {})
+        account = KillBillClient::Model::Account.new(:account_id => account_id, :is_notified_for_invoices => is_notified)
+
+        account.update_email_notifications(user, reason, comment, options)
```

Comment:
This won't work anymore with Kill Bill 0.20.x. Next step is probably to enhance the email-notifications plugin to add support for such configuration. See https://github.com/killbill/killbill-email-notifications-plugin/issues/8.

Once this is done, we can update this to call the new plugin servlet endpoint instead, similar to what we do with the Analytics plugin (https://github.com/killbill/killbill-analytics-ui/blob/master/app/helpers/kanaui/dashboard_helper.rb).